parking_permits/customer_permit.py

```diff
@@ -55,12 +55,18 @@ def __init__(self, customer_id):
 
     def get(self):
         permits = []
-        # Update the start_time to next day for all the draft permits
+        # Delete all the draft permits if it wasn't created today
+        self.customer_permit_query.filter(
+            status=DRAFT, start_time__lt=tz.now()
+        ).delete()
+
         for permit in self.customer_permit_query.order_by("start_time"):
-            if permit.status == DRAFT:
-                permit.start_time = next_day()
-                permit.end_time = get_end_time(next_day(), permit.month_count)
-                permit.save(update_fields=["start_time", "end_time"])
+            vehicle = permit.vehicle
+            # Update vehicle detail from traficom if it wasn't updated today
+            if permit.vehicle.updated_from_traficom_on < tz.now().date():
```

Comment:
Minor, but if it's Finland local time that we are getting from Traficom, I think we should compare it with local date also. `timezone.localdate(timezone.now())`